app/components/shared/ExternalLink.js

```diff
@@ -1,15 +1,20 @@
 import { shell } from "electron";
 import { network } from "connectors";
+import { Button } from "pi-ui";
 
 const clicker = (isTestNet, href, hrefTestNet) => () => {
   const url = hrefTestNet && isTestNet ? hrefTestNet : href;
   shell.openExternal(url);
 };
 
-const ExternalLink = ({ className, href, children, hrefTestNet, isTestNet }) => (
-  <a className={className} onClick={clicker(isTestNet, href, hrefTestNet) } href="#">
-    {children}
-  </a>
+const ExternalLink = ({ className, href, children, hrefTestNet, isTestNet, isButton }) => (
```

Comment:
I believe we can have only one tag here. No need to have the option to this component being a button or a link.

Just make sure it work in all places we use it (it should I believe)

Reply:
It needed here in order to supper normal links (e.g the example in the following screenshot) - we could make use pi-ui's link here as well tho - I will give a try.

![image](https://user-images.githubusercontent.com/10324528/79518353-2d5d0a80-8051-11ea-87fe-82c6b4a2ce38.png)

Reply:
Here I'll suggest using the Link component from pi-ui. It does have a customComponent property where you can pass the Button component e.g `<Link href="#" customComponent={() => <Button>Click me</Button>} />`. I added it to the docs here: https://github.com/decred/pi-ui/pull/250

Reply:
With the above comment, you can probably remove the `isButton` flag. I'm not a big fan of the `isSomething` flag when it's not state related. I'm making an effort to remove them from pgui as well. You could use the same behavior as the Link component and pass a render component to ExternalLink instead. if the component is used multiple times you can create a ExternalLinkAsButton component and it will be basically `<ExternalLink renderComponent={() => <Button ... />}/>`. IMO the code becomes more declarative this way.

Reply:
Great. I am not really a fan of overcomplicating components as well. Glad you pointed the directions so amass can deal with it :)

Reply:
@amassarwi can you add tiago's comment?

Reply:
Sure, will do!

Reply:
Done, Instead of changing `ExternalLink` which is used in many other parts of the APP, I have refactored `PoliteiaLink` to not rely on `useExternalLink`, and to use combination of `pi-ui`'s `Link` & `Component` depends on the usage.